src/main.rs

```diff
@@ -616,6 +635,21 @@ mod tests {
                 .is_empty())
         }
 
+        fn get_win_size(&self) -> (u16, u16) {
+            let col = self
+                .ssh_command("echo $COLUMNS >  winsize; cat winsize")
```

Comment:
Why do you have to redirect it to a file only to cat it?

Reply:
I just realized that I can just use pipe. Thanks.